elasticluster/cluster.py

```diff
@@ -740,6 +744,47 @@ def stop(self, force=False, wait=False):
         else:
             self._delete_saved_data()
 
+    def pause(self):
+        """Pause all VMs in this cluster and store data so that they
+        can be restarted later.
+        """
+        log.debug("Pausing cluster `%s`", self.name)
+        failed = self._pause_all_nodes()
+        if os.path.exists(self.known_hosts_file):
+            os.remove(self.known_hosts_file)
+        self.repository.save_or_update(self)
+        if failed:
+            log.warning(
+                    "Not all cluster nodes have been successfully "
+                    "stopped.  Some nodes may still be running - "
+                    "check error messages above and consider "
+                    "re-running `elasticluster pause %s` if "
+                    "necessary.", self.name)
+
+    def resume(self):
+        """
+        Resume all paused VMs in this cluster.
+        """
+        log.debug("Resuming cluster `%s`", self.name)
```

Comment:
I'd raise this to severity `info`:
```suggestion
        log.info("Resuming cluster `%s` ...", self.name)
```